flow/e2e/congen.go

```diff
@@ -137,7 +138,15 @@ func SetupPostgres(suffix string) (*pgxpool.Pool, error) {
 func TearDownPostgres(pool *pgxpool.Pool, suffix string) error {
 	// drop the e2e_test schema
 	if pool != nil {
-		err := cleanPostgres(pool, suffix)
+		deadline := time.Now().Add(time.Minute)
```

Comment:
I was unable to figure out how to make `ExecuteWorkflow` wait on activity stopping (not just cancelling), I was unable to figure out any means to poll for activities, but immediately getting to teardown while consumeStream is still active would fail over replication slot still being in use so it can't be dropped

Solution: `WaitFor` everywhere. teardown has a 1 minute retry loop